src/main/java/com/minecolonies/coremod/entity/citizen/EntityCitizen.java

```diff
@@ -423,7 +425,25 @@ public InteractionResult checkAndHandleImportantInteractions(final Player player
      */
     private InteractionResult directPlayerInteraction(final Player player, final InteractionHand hand)
     {
+        if (player.isShiftKeyDown())
+        {
+            return null;
+        }
+
+        // todo remove before release
         final ItemStack usedStack = player.getItemInHand(hand);
+        if (usedStack.getItem() instanceof BlockItem && ((BlockItem) usedStack.getItem()).getBlock() instanceof AbstractBlockHut<?>)
```

Comment:
This is temp code for easy testing, will be removed after testing is over. Reminder here